docs-src/go/how-to-set-an-update-validator-function-in-go.md

```diff
@@ -0,0 +1,50 @@
+---
+id: how-to-set-an-update-validator-function-in-go
+title: How to set an Update validator function in go
+sidebar_label: Validator function
+description: Use the SetUpdateHandlerWithOptions API and pass it a validator function to validate inputs.
+---
+
+Validate certain aspects of the data sent to the Workflow using an Update validator function.
+For instance, a counter Workflow might never want to accept a non-positive number.
+Invoke the `SetUpdateHandlerWithOptions` API and define a validator function as one of the options.
+
+When you use a Validator function, the Worker receives the Update first, before any Events are written to the Event History.
+If the Update is rejected, it's not recorded in the Event History.
+If it's accepted, the `WorkflowExecutionUpdateAccepted` Event occurs.
```

Comment:
Can you add something along the lines of "A panic in the Validator is treated as a rejection of the Update"